utils.sh

```diff
@@ -334,9 +334,9 @@ function generate_metal3_config {
 }
 
 function image_mirror_config {
-    if [ ! -z "${MIRROR_IMAGES}" || ! -z "${ENABLE_LOCAL_REGISTRY}" ]; then
+    if [[ ! -z "${MIRROR_IMAGES}" || ! -z "${ENABLE_LOCAL_REGISTRY}" ]]; then
```

Comment:
Looks like the same problem could also be here: https://github.com/openshift-metal3/dev-scripts/blob/41b9cfb172af92788ab9c3395b2d60ae90f886f0/ocp_install_env.sh#L136?

Reply:
I think so